### PR TITLE
[SYCLomatic]Fix the bug some member of cudaDeviceProp would not be migrated

### PR DESCRIPTION
**Pull request by wenhuiNi**
The member of `CudaDeviceProp` should be initiated by `cudaGetDeviceProperties`.
It is unreasonable to directly assign values to its member variables

### CODE REVIEW

**Comment by tangjj11** [comment]
Why use "would" in title? 
Suggestion: Support (how many) members of 'cudaDeviceProp' ......

**Comment by wenhuiNi** [comment]
>@tangjj11 'Why use "would" in title?'

because in some case, some member of cudaDeviceProp will be migrated.

**Review by AndyCHHuang — APPROVED** [review]
LGTM

File: clang/test/dpct/test_api_level/Device/api_test26.cu [hunk 1]
```diff
@@ -1,40 +1,29 @@
...
-  deviceProp.memoryBusWidth = 1;
+  cudaGetDeviceProperties(&deviceProp, 0);
+  deviceProp.uuid;
+  deviceProp.memoryClockRate;
+  deviceProp.memoryBusWidth;
+  deviceProp.minor;
+  deviceProp.multiProcessorCount;
   return 0;
-}
+}
```
Comment (yihwang-nv) [inline clang/test/dpct/test_api_level/Device/api_test26.cu:39]: Please add new line at the end of this file.

**Review by ziranzha — APPROVED** [review]

**Pull request merged**